data/data-pipeline/data_pipeline/tests/score/fixtures.py

```diff
@@ -1,13 +1,217 @@
 import pandas as pd
 import pytest
 from data_pipeline.config import settings
-from data_pipeline.score import field_names
+from data_pipeline.score.field_names import GEOID_TRACT_FIELD
+from data_pipeline.etl.score import constants
 
 
 @pytest.fixture(scope="session")
 def final_score_df():
     return pd.read_csv(
         settings.APP_ROOT / "data" / "score" / "csv" / "full" / "usa.csv",
```

Comment:
Question: will this test fail until the user has run the full score pipeline at least once on their local? 

If so, that could potentially make it hard for new developers who are just "opening the box" on the CEJST.

Reply:
It will, but any tests that use it are marked as smoketest and skipped by default. I should add something to the readme tho, so I will do so.